<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
    <meta name="description" content="BioFolD - Manuscripts">
    <meta name="author" content="BioFolD - Emidio Capriotti">
    <link rel="icon" href="/favicon.ico">

    <title>BioFolD - Manuscripts</title>

    <!-- Bootstrap core CSS -->
    <link href="pages/dist/css/bootstrap.min.css" rel="stylesheet">

    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug 
    <link href="pages/assets/css/ie10-viewport-bug-workaround.css" rel="stylesheet">
    -->

    <!-- Custom styles for this template -->
    <link href="pages/style/jumbotron-narrow-biofold.css" rel="stylesheet">

    <!-- Just for debugging purposes. Don't actually copy these 2 lines! -->
    <!--[if lt IE 9]><script src="../../assets/js/ie8-responsive-file-warning.js"></script><![endif]
    <script src="pages/assets/js/ie-emulation-modes-warning.js"></script>
    -->
    <!-- Altmetric -->
    <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
    <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>

  </head>

  <body>
    <br>
    <div class="container">

      <div class="jumbotron">
        <table width="100%">
        <tr>
        <td width="90%">
        <div class='biofold' ><span style="color:#c31e27;">Bio</span><span style="color:#4689a3;">molecules</span> <span style="color:#c31e27;">Fol</span><span style="color:#4689a3;">ding and </span><span style="color:#c31e27;">D</span><span style="color:#4689a3;">isease</span></div><br>
	</td>
	<td width="10%">
	<div class="right_cell">
        <a href="http://biofold.org">
        <img src="pages/img/logo.png" width="75px" align="left">
        </a>
        </div>
	</td>
	</tr>
	</table>

	<table width="100%">
        <tr>
        <td>
        <div class="form-inline justify-content-center">
        <nav>
          <ul class="nav nav-pills">
            <li role="presentation" ><a href="index.html">Home</a></li>
            <li role="presentation"><a href="research.html">Research</a></li>
            <li role="presentation" ><a href="people.html">People</a></li>
            <li role="presentation"><a href="resources.html">Resources</a></li>
            <li role="presentation" class='active'><a href="papers.html">Publications</a></li>
            <li role="presentation"><a href="training.html">Training</a></li>
          </ul>
        </nav>
        </div>
        </td>
        </tr>
        </table>
    <br>
  	<br>
  	
     <div class="text">
     <h4 class="title" >Papers</h4>
     </div>
     <div class="text">
     <p class="text">
     Since 2004 we have published 63 peer-reviewed manuscripts (11 reviews) in international journal with impact factor and 6 book chapters.
     The complete list of research articles and book chapters is reported below.
     <br><br>

     <div class="text">
     <h4 class="section">
     2022
     </h4></div>
     <table cellpadding="7" cellspacing="7" class="paper">
     <tbody><tr><td>
     <ol>

<li class="text" value=63>
     <p class='text-justify'>
     <i>Montanucci L<sup>&dagger;</sup>, <span style="font-weight:500;">Capriotti E<sup>&dagger;</sup></span>, Birolo G, Benevenuta S, Pancotti C, Lal D, Fariselli P</i> (2022).
    DDGun: an untrained predictor of protein stability changes upon amino acid variants.
     <b>Nucleic Acids Research</b>. DOI:10.1093/nar/gkac325.<br>
     <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/"><img src="pages/img/pubmed.gif"></a>
     <a href="https://academic.oup.com/nar/advance-article-pdf/doi/10.1093/nar/gkac325/43614328/gkac325.pdf"><img width="15" src="pages/img/pdf.gif"></a>
      <a href="pages/documents/papers/gkac325-supmat.pdf">
     <img width="15" src="pages/img/doc.gif"></a>
     <a href="pages/documents/papers/ddgun-preds.tar.gz">
     <img width="15" src="pages/img/zip.gif"></a>
     <a target=blank href="https://folding.biofold.org/ddgun"><img width="16" src="pages/img/webserver.jpg"></a>
     <a target=blank href="https://github.com/biofold/ddgun"><img width="20" src="pages/img/github.png"></a>
     <a target=blank href="https://hub.docker.com/r/biofold/ddgun"><img width="24" src="pages/img/docker.png"></a>
     &nbsp;
      <span class="__dimensions_badge_embed__" data-doi="10.1093/nar/gkac325" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
      <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1093/nar/gkac325" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br><br>


<li class="text" value=62>
     <p class='text-justify'>
      <i><span style="font-weight:500;">Capriotti E*</span>, Fariselli P*</i> (2022).
     Evaluating the relevance of sequence conservation in the prediction of pathogenic missense variants.
     <b>Human Genetics</b>. DOI:10.1007/s00439-021-02419-4.<br>
     <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/35098354"><img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/capriotti-huge2022.pdf"><img width="15" src="pages/img/pdf.gif"></a>
     &nbsp;
      <span class="__dimensions_badge_embed__" data-doi="10.1007/s00439-021-02419-4" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
      <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1007/s00439-021-02419-4" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br><br>


<li class="text" value=61>
     <p class='text-justify'>
      <i>Pancotti C, Benevenuta S, Birolo G, Repetto V, Alberini V, Sanavia T, <span style="font-weight:500;">Capriotti E*</span>, Fariselli P* </i> (2022).
     Predicting protein stability changes upon single-point mutation: a thorough comparison of the available tools on a new dataset.
     <b>Briefings in Bioinformatics</b>. DOI:10.1093/bib/bbab555<br>
     <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/35021190"><img src="pages/img/pubmed.gif"></a>
     <a href="https://academic.oup.com/bib/advance-article-pdf/doi/10.1093/bib/bbab555/42128432/bbab555.pdf?guestAccessKey=a54dbef8-b7d0-495f-965a-1d315c4f0055"><img width="15" src="pages/img/pdf.gif"></a>
     &nbsp;
     <span class="__dimensions_badge_embed__" data-doi="10.1093/bib/bbab555" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
      <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1093/bib/bbab555" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br><br>

     </ol>
     </td></tr></tbody></table>

<div class="text">
     <h4 class="section">
     2021
     </h4></div>
     <table cellpadding="7" cellspacing="7" class="paper">
     <tbody><tr><td>
     <ol>


<li class="text" value=60>
     <p class='text-justify'>
     <i>
     Walsh I, Fishman D, Garcia-Gasulla D, Titma T, Pollastri G,<span style="font-weight:500;"> Capriotti E</span> , 
     Casadio R, Capella-Gutierrez S, 
     Cirillo D, Del Conte A, Dimopoulos AC, Del Angel V. D, Dopazo J, Fariselli P, Fernandez J. M, Huber F, 
     Kreshuk A, Lenaerts T, Martelli PL, Navarro A, Broin PO, Pinero J, Piovesan D, Reczko M, Ronzano F, 
     Satagopam V, Savojardo C, Spiwok V, Tangaro MA, Tartari G, Salgado D, Valencia A, Zambelli F, Harrow J, 
     Psomopoulos FE, Tosatto SCE.
     </i> (2021).
     DOME: recommendations for supervised machine learning validation in biology.
     <b>Nature Methods</b>. DOI:10.1038/s41592-021-01205-4.<br>
     <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/34316068"><img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/walsh_nmeth2021.pdf"><img width="15" src="pages/img/pdf.gif"></a>
     &nbsp;
      <span class="__dimensions_badge_embed__" data-doi="10.1038/s41592-021-01205-4" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
      <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1038/s41592-021-01205-4" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br><br>

<li class="text" value=59>
    <p class='text-justify'>
    <i>Merlotti A, Menichetti G, Fariselli P, <span style="font-weight:500;">Capriotti E</span>, Remondini D.</i> (2021).
    Network-based strategies for protein characterization.
    <b>Advances in Protein Chemistry and Structural Biology</b>. 127: 217-248. DOI:10.1016/bs.apcsb.2021.05.001.
    <br><a target=blank href="https://www.sciencedirect.com/science/article/pii/S1876162321000511">
    <img src="pages/img/abstract.gif"></a>
    <a href="pages/documents/papers/merlotti-apcsb2021.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
    &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1016/bs.apcsb.2021.05.001" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1016/bs.apcsb.2021.05.001" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br><br>

<li class="text" value=58>
     <p class='text-justify'>
     <i> Pancotti C, Benevenuta S, Repetto V, Birolo G, <span style="font-weight:500;">Capriotti E</span>, Sanavia T, Fariselli P.</i> (2021).
     A deep-learning sequence-based method to predict protein stability changes upon genetic variations.
     <b>Genes</b>. 12:911. DOI:10.3390/genes12060911.<br>
     <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/34204764"><img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/pancotti-genes2021.pdf"><img width="15" src="pages/img/pdf.gif"></a>
     &nbsp;
      <span class="__dimensions_badge_embed__" data-doi="10.3390/genes12060911" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
      <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.3390/genes12060911" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br><br>


<li class="text" value=57>
     <p class='text-justify'>
     <i> Petrosino M, Novak L, Pasquo A, Chiaraluce R, Turina P, <span style="font-weight:500;">Capriotti E*</span>, Consalvi V*.</i> (2021).
     Analysis and interpretation of the impact of missense variants in cancer.
     <b>International Journal of Molecular Sciences</b>. 22:5416. DOI:10.3390/ijms22115416.<br>
     <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/34063805"><img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/petrosino-ijms2021.pdf"><img width="15" src="pages/img/pdf.gif"></a>
     <a href="pages/documents/papers/petrosino-ijms2021-supmat.pdf">
     <img width="15" src="pages/img/doc.gif"></a>
      <a href="pages/documents/papers/supplementary_file1.tsv.gz">
     <img width="15" src="pages/img/zip.gif"></a>
     &nbsp;
      <span class="__dimensions_badge_embed__" data-doi="10.3390/ijms22115416" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
      <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.3390/ijms22115416" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br><br>


<li class="text" value=56>
     <p class='text-justify'>
     <i>Turina P, Fariselli P*,  <span style="font-weight:500;">Capriotti E*</span></i>. (2021).
     ThermoScan: Semi-automatic identification of protein stability data from PubMed.
     <b>Frontiers in Molecular Biosciences</b>.8:620475. DOI:10.3389/fmolb.2021.620475.
      <br>
     <a target=blank href="https://pubmed.ncbi.nlm.nih.gov/33842537"><img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/turina_fmbs2021.pdf"><img width="15" src="pages/img/pdf.gif"></a>
     <a href="pages/documents/papers/turina_fmbs2021_supmat.pdf">
     <img width="15" src="pages/img/doc.gif"></a>
     <a href="pages/documents/papers/thermoscan-supfiles.tar.gz">
     <img width="15" src="pages/img/zip.gif"></a>
     <a target=blank href="https://folding.biofold.org/thermoscan"><img width="18" src="pages/img/webserver.jpg"></a>
     <a target=blank href="https://github.com/biofold/ThermoScan"><img width="20" src="pages/img/github.png"></a>
     &nbsp;
     <span class="__dimensions_badge_embed__" data-doi="10.3389/fmolb.2021.620475" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
     <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.3389/fmolb.2021.620475" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
     <br><br>


<li class="text" value=55>
     <p class='text-justify'>
     <i>Birolo G, Benevenuta S, Fariselli P*, <span style="font-weight:500;">Capriotti E*</span>, 
        Giorgio E, Sanavia T</i>. (2021).
     Protein stability perturbation contributes to the loss of function in haploinsufficient genes.
     <b>Frontiers in Molecular Biosciences</b>. 8:620793. DOI:10.3389/fmolb.2021.620793.
      <br>
     <a target=blank href="https://pubmed.ncbi.nlm.nih.gov/33598480"><img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/birolo_fmolb2021.pdf"><img width="15" src="pages/img/pdf.gif"></a>
     <a href="pages/documents/papers/">
     <img width="15" src="pages/img/doc.gif"></a>
     &nbsp;
      <span class="__dimensions_badge_embed__" data-doi="10.3389/fmolb.2021.620793" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
     <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.3389/fmolb.2021.620793" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
     <br><br>


     </ol>
     </td></tr></tbody></table>

     <div class="text">
     <h4 class="section">
     2020
     </h4></div>
     <table cellpadding="7" cellspacing="7" class="paper">
     <tbody><tr><td>
     <ol>
    
     <li class="text" value=54>
     <p class='text-justify'>
     <i>Benevenuta  S, <span style="font-weight:500;">Capriotti E*</span>, Fariselli P*</i>. (2020).
     Calibrating variant-scoring methods for clinical decision making.
     <b>Bioinformatics</b>. 36:5709-5711.
      <br>
     <a target=blank href="https://pubmed.ncbi.nlm.nih.gov/33492342"><img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/benevenuta-bioinfo2020.pdf"><img width="15" src="pages/img/pdf.gif"></a>
     <a href="pages/documents/papers/benevenuta-bioinfo2020-supmat.pdf">
     <img width="15" src="pages/img/doc.gif"></a>
     &nbsp;
      <span class="__dimensions_badge_embed__" data-doi="10.1093/bioinformatics/btaa943" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
     <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1093/bioinformatics/btaa943" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
     <br><br>     

     <li class="text" value=53>
     <p class='text-justify'>     
     <i>Sanavia T, Birolo G, Montanucci L, Turina P, <span style="font-weight:500;">Capriotti E*</span>, Fariselli P*</i>. (2020).
     Limitations and challenges in protein stability prediction upon genome varia‐tions: towards future applications in precision medicine.
     <b>Computational and Structural Biotechnology Journal</b>. 18: 1968-1079.
      <br>
     <a target=blank href="https://pubmed.ncbi.nlm.nih.gov/32774791/"><img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/sanavia_csbj2020.pdf"><img width="15" src="pages/img/pdf.gif"></a>
     &nbsp;
      <span class="__dimensions_badge_embed__" data-doi="10.1016/j.csbj.2020.07.011" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
     <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1016/j.csbj.2020.07.011" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
     <br><br>
     </ol>
     </td></tr></tbody></table>


     <div class="text">
     <h4 class="section">
     2019
     </h4></div>
     <table cellpadding="7" cellspacing="7" class="paper">
     <tbody><tr><td>
     <ol>
     <li class="text" value=52>
     <p class='text-justify'>
     <i>Kasak L, Bakolitsa C, Hu Z, Yu C, Rine J, Dimster-Denk DF, Pandey G, De Baets  G, Bromberg Y, Cao C,      <span style="font-weight:500;">Capriotti E</span>, 
      Casadio R, Van Durme J, Giollo M, Karchin R,
      Katsonis P, Leonardi E, Lichtarge O, Martelli PL, Masica D, Mooney SD, Olatubosun A, 
      Pal LR, Radivojac P, Rousseau F, Savojardo C, Schymkowitz J, Thusberg J,
      Tosatto SCE, Vihinen M, Väliaho J, Repo S, Moult J, Brenner SE, Friedberg I.</i> (2019).
      Assessing Computational Predictions of the Phenotypic Effect of
      Cystathionine-beta-Synthase Variants. 
      <b>Human Mutation</b>. 40: 1530-1545.
      <br>
     <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/31301157"><img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/Kasak_et_al-2019-Human_Mutation.pdf"><img width="15" src="pages/img/pdf.gif"></a>
     &nbsp;
      <span class="__dimensions_badge_embed__" data-doi="10.1002/humu.23868" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
     <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1002/humu.23868" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br> 
     <br>      

       <li class="text" value=51>
     <p class='text-justify'>
     <i>Zhang J, Kinch LN, Cong Q, Katsonis P, Lichtarge O, Savojardo C, Babbi G, Martelli PL, 
        <span style="font-weight:500;">Capriotti E</span>, 
        Casadio R, Garg A, Pal D, Weile J, Sun S, Verby M, Roth FP, Grishin NV. </i> (2019)
        Assessing predictions on fitness effects of missense variants in calmodulin. 
        <b>Human Mutation</b>. 40: 1463-1473.  
        <br>
     <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/31283071"><img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/Zhang_et_al-2019-Human_Mutation.pdf"><img width="15" src="pages/img/pdf.gif"></a>
     &nbsp;
      <span class="__dimensions_badge_embed__" data-doi="10.1002/humu.23857" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
     <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1002/humu.23857" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
     <br>
     <br>
    
      <li class="text" value=50>
     <p class='text-justify'>
     <i>Monzon AM, Carraro M, Chiricosta L, Reggian F, Han J, Ozturk K, Wang Y, Miller M, Bromberg Y, 
     <span style="font-weight:500;">Capriotti E</span>, 
     Savojardo C, Babbi G, Martelli PL, Casadio R, Katsonis P, Lichtarge O, Carter H, 
     Kousi M, Katsanis N, Andreoletti G, Moult J, Brenner SE, Ferrari C, Leonardi E, Tosatto SCE. </i>
     (2019). Performance of computational methods for the evaluation of Pericentriolar Material 1 
     missense variants in CAGI-5.
     <b>Human Mutation</b>. 40: 1474-1485.
     <br>
     <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/31260570"><img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/Monzon_et_al-2019-Human_Mutation.pdf"><img width="15" src="pages/img/pdf.gif"></a>
     &nbsp;
      <span class="__dimensions_badge_embed__" data-doi="10.1002/humu.23856" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
     <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1002/humu.23856" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
     <br>
      
     <li class="text" value=49>
     <p class='text-justify'>
     <i>Voskanian A, Katsonis P, Lichtarge O, Pejaver V, Radivojac P, Mooney SD,
      <span style="font-weight:500;">Capriotti E</span>,
     Bromberg Y, Wang Y, Miller M, Martelli PL, Savojardo C, Babbi G, Casadio R, Cao Y, Sun Y, Shen Y, Garg A, Pal D, Yu Y, Huff CD, 
     Tavtigian SV, Young E, Neuhausen SL, Ziv E, Pal LR, Andreoletti G, Brenner S, Moult J, Kann MG.</i> (2019). 
     Assessing the performance of in-silico methods for predicting the pathogenicity of variants in the gene CHEK2, 
     among Hispanic females with breast cancer. 
     <b>Human Mutation</b>. 40: 1612-1622.
     <br>
     <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/31241222"><img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/Voskanian_et_al-2019-Human_Mutation.pdf"><img width="15" src="pages/img/pdf.gif"></a>
     &nbsp;
      <span class="__dimensions_badge_embed__" data-doi="10.1002/humu.23849" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
     <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1002/humu.23849" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
     <br>

     <li class="text" value=48>
     <p class='text-justify'>
     <i> Savojardo C<sup>&dagger;</sup>, Petrosino M<sup>&dagger;</sup>, Babbi G, Bovo S, Corbi-Verge C, Casadio R, Piero Fariselli P, 
        Folkman L, Garg A, Karimi M, Katsonis P, Kim PM, Lichtarge O, Martelli PL, Pasquo A, Pal D, 
        Shen Y, Strokach AV, Turina P, Zhou Y, Andreoletti G, Brenner S, Chiaraluce R, Consalvi V, 
        <span style="font-weight:500;">Capriotti E*.</span></i> (2019).
     Evaluating the predictions of the protein stability change upon single amino acid substitutions for the FXN CAGI5 challenge.
     <b>Human Mutation</b>. 40: 1392-1399.
     <br>
     <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/31209948"><img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/Savojardo-Petrosino_et_al-2019-Human_Mutation.pdf"><img width="15" src="pages/img/pdf.gif"></a>
     <a href="pages/documents/papers/Savojardo-Petrosino_et_al-2019-Human_Mutation-supmat.pdf">
     <img width="15" src="pages/img/doc.gif"></a>
     &nbsp;
      <span class="__dimensions_badge_embed__" data-doi="10.1002/humu.23843" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
      <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1002/humu.23843" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
     <br>

     <li class="text" value=47>
     <p class='text-justify'>
     <i>Montanucci L, <span style="font-weight:500;">Capriotti E*</span>, Frank Y, Ben-Tal N, Fariselli P*.</i> (2019).
     DDGun: an untrained method for the prediction of protein stability changes upon single and multiple point variations. 
     <b>BMC Bioinformatics</b>. 20 (Suppl 14): 335.
     <br>
     <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/31266447"><img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/Montanucci_et_al-2019-BMC_Bioinformatics.pdf"><img width="15" src="pages/img/pdf.gif"></a>
     <a href="http://folding.biofold.org/ddgun">
     <img width="15" src="pages/img/zip.gif"></a>
     &nbsp;
      <span class="__dimensions_badge_embed__" data-doi="10.1186/s12859-019-2923-1" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
      <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1186/s12859-019-2923-1" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
     <br>

     <li class="text" value=46>
     <p class='text-justify'>
     <i>McInnes G, Daneshjou R, Katsonis P, Lichtarge O, Srinivasan RG, Rana S, Radivojac P, Mooney SD, Pagel KA, Stamboulian M, Jiang Y, <span style="font-weight:500;">Capriotti E</span>, Wang Y, Bromberg Y, Bovo S, Savojardo C, Martelli PL, Casadio R, Pal LR, Moult J, Brenner S, Altman R. </i> (2019).
     Predicting venous thromboembolism risk from exomes in the Critical Assessment of Genome Interpretation (CAGI) challenges. 
     <b>Human Mutation</b>. 40: 1314-1320.
     <br>
     <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/31140652"><img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/McInness_HumanMutation2019.pdf"><img width="15" src="pages/img/pdf.gif"></a>
     <a href="pages/documents/papers/McInness_HumanMutation2019-supmat.pdf">
     <img width="15" src="pages/img/doc.gif"></a>
     &nbsp;
      <span class="__dimensions_badge_embed__" data-doi="10.1002/humu.23825" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
      <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1002/humu.23825" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
     <br>

     <li class="text" value=45>
     <p class='text-justify'>
     <i>
     Petrosino M, Pasquo A, Novak L, Toto A, Gianni S, Mantuano E, Veneziano L, Minicozzi V, Pastore A, Puglisi R,
     <span style="font-weight:500;">Capriotti E</span>, Chiaraluce R, Consalvi V. </i> (2019).
     Characterization of human frataxin missense variants in cancer tissues.
     <b>Human Mutation</b>. 40: 1400-1413.
     <br>
     <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/31074541"><img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/Petrosino_et_al-2019-Human_Mutation.pdf"><img width="15" src="pages/img/pdf.gif"></a>
     <a href="pages/documents/papers/Petrosino_et_al-2019-Human_Mutation-supmat.pdf">
     <img width="15" src="pages/img/doc.gif"></a>
     &nbsp;
      <span class="__dimensions_badge_embed__" data-doi="10.1002/humu.23789" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
      <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1002/humu.23789" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
     <br>

     <li class="text" value=44>
     <p class='text-justify'>
     <i>Savojardo C, Babbi G, Bovo S, <span style="font-weight:500;">Capriotti E</span>, Martelli PL, Casadio R.</i> (2019).
     Are machine learning based methods suited to address complex biological problems? Lessons from CAGI‐5 challenges
     <b>Human Mutation</b>. 40: 1455-1462
     <br>
     <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/31066146"><img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/Savojardo_et_al-2019-Human_Mutation.pdf"><img width="15" src="pages/img/pdf.gif"></a>
     <a href="pages/documents/papers/Savojardo_et_al-2019-Human_Mutation-supmat.pdf">
     <img width="15" src="pages/img/doc.gif"></a> 
      &nbsp;
      <span class="__dimensions_badge_embed__" data-doi="10.1002/humu.23784" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
      <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1002/humu.23784" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
     <br>

     <li class="text" value=43>
     <p class='text-justify'>
     <i><span style="font-weight:500;">Capriotti E<sup>&dagger;</sup>*</span>, Montanucci L<sup>&dagger;</sup>, Profiti G, Rossi I, Giannuzzi D, Aresu L, Fariselli P.</i> (2019).
     Fido-SNP: The first webserver for scoring the impact of single nucleotide variants in the dog genome.
     <b>Nucleic Acids Research</b>. 47(W1): W136-W141.
     <br>
     <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/31114899"><img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/gkz420.pdf"><img width="15" src="pages/img/pdf.gif"></a>
     <a href="pages/documents/papers/gkz420-supmat.pdf">
     <img width="15" src="pages/img/doc.gif"></a>
      <a href="pages/documents/papers/fido-snp-supfiles.tar.gz">
     <img width="15" src="pages/img/zip.gif"></a>
     <a target=blank href="https://snps.biofold.org/fido-snp"><img width="18" src="pages/img/webserver.jpg"></a>
     <a target=blank href="https://github.com/biofold/Fido-SNP"><img width="20" src="pages/img/github.png"></a>
     <a target=blank href="https://hub.docker.com/r/biofold/fido-snp"><img width="24" src="pages/img/docker.png"></a>
     &nbsp;
      <span class="__dimensions_badge_embed__" data-doi="10.1093/nar/gkz420" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
      <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1093/nar/gkz420" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
     <br>

     <li class="text" value=42>
     <p class='text-justify'>
          <i><span style="font-weight:500;">Capriotti E*</span>, Ozturk K, Carter H*.</i> (2019).
	Integrating molecular network with genetic variant interpretation for precision medicine.
        <b>WIREs Systems Biology and Medicine.</b>
        11: e1443.<br>
        <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/30548534"><img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/capriotti_wsbm2018.pdf"><img width="15" src="pages/img/pdf.gif"></a>
     &nbsp;
      <span class="__dimensions_badge_embed__" data-doi="10.1002/wsbm.1443" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
      <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1002/wsbm.1443" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
     <br>
     </ol>
     </td></tr></tbody></table>

     <div class="text">
     <h4 class="section"> 
     2017
     </h4></div>
     <table cellpadding="7" cellspacing="7" class="paper">
     <tbody><tr><td>
     <ol>
     <li class="text" value=41>
     <p class='text-justify'>
     <i><span style="font-weight:500;">Capriotti E*</span>, Fariselli P*.</i> (2017).
     PhD-SNP<sup>g</sup>: A webserver and lightweight tool for scoring single nucleotide variants.
     <b>Nucleic Acids Research.</b> 45: W247-W252.<br>
     <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/28482034">
     <img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/phdsnpg-web-2017.pdf">
     <img width="15" src="pages/img/pdf.gif"></a>
     <a href="pages/documents/papers/phdsnpg-web-2017-supmat.pdf">
     <img width="15" src="pages/img/doc.gif"></a>
     <a href="pages/documents/papers/phdsnpg-web-2017-supfiles.gz">
     <img width="15" src="pages/img/zip.gif"></a>
     <a target=blank href="https://snps.biofold.org/phd-snpg"><img width="18" src="pages/img/webserver.jpg"></a>
     <a target=blank href="https://github.com/biofold/PhD-SNPg"><img width="20" src="pages/img/github.png"></a>
     <a target=blank href="https://hub.docker.com/r/biofold/phd-snpg"><img width="24" src="pages/img/docker.png"></a>
      &nbsp; 
      <span class="__dimensions_badge_embed__" data-doi="10.1093/nar/gkx369" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
      <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1093/nar/gkx369" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
     <br>

     <li class="text" value=40>
     <p class='text-justify'>
     <i>Carraro M, Minervini G, Giollo M, Bromberg Y, <span style="font-weight:500;">Capriotti E</span>, Casadio R, Dunbrack R, Elefanti L,
     Fariselli P, Ferrari C, Gough J, Katsonis P, Leonardi E, Lichtarge O, Menin C, Martelli PL, Niroula A, 
     Pal LR, Repo S, Scaini MC, Vihinen M, Wei Q, Xu Q, Yang Y, Yin Y, Zaucha J, Zhao H, Zhou Y, Brenner SE, 
     Moult J, Tosatto SCE.</i> (2017). 
     Performance of in silico tools for the evaluation of p16INK4a (CDKN2A) variants in CAGI. 
     <b>Human Mutation.</b> 38: 1042-1050.<br>
     <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/28440912"><img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/Carraro_HumMut2017.pdf"><img width="15" src="pages/img/pdf.gif"></a>
     <a href="pages/documents/papers/Carraro_HumMut2017_SupMat.pdf">
     <img width="15" src="pages/img/doc.gif"></a>
     &nbsp;
      <span class="__dimensions_badge_embed__" data-doi="10.1002/humu.23235" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
      <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1002/humu.23235" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
     <br>

     <li class="text" value=39>
     <p class='text-justify'>
     <i><span style="font-weight:500;">Capriotti E*</span>, Martelli PL, Fariselli P, Casadio R.</i> (2017).
     Blind prediction of deleterious amino acid variations with SNPs&GO.
     <b>Human Mutation.</b> 38: 1064-1071.<br>
     <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/28102005"><img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/Capriotti_HumMut2017.pdf"><img width="15" src="pages/img/pdf.gif"></a>
     <a href="pages/documents/papers/Capriotti_HumMut2017_SupMat.pdf">
     <img width="15" src="pages/img/doc.gif"></a>
     &nbsp;
      <span class="__dimensions_badge_embed__" data-doi="10.1002/humu.23179" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
      <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1002/humu.23179" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
     <br></ol>
     </td></tr></tbody></table>
     
     <div class="text">
     <h4 class="section"> 
     2016
     </h4></div>
     <table cellpadding="7" cellspacing="7" class="paper">
     <tbody><tr><td>
     <ol>
     <li class="text" value=38>
     <p class='text-justify'>
     <i>Khass M, Blackburn T, Burrows PD, Walter MR, <span style="font-weight:500;">Capriotti E</span>, Schroeder HW Jr.</i> (2016). 
	VpreB serves as an invariant surrogate antigen for selecting immunoglobulin antigen-binding sites. <b>Science Immunology.</b> 
        1: aaf6628.<br>
	<a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/28217764"><img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/khass_scienceimmunology2016.pdf"><img width="15" src="pages/img/pdf.gif"></a>
     &nbsp;
      <span class="__dimensions_badge_embed__" data-doi="10.1126/sciimmunol.aaf6628" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
      <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1126/sciimmunol.aaf6628" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
     <br></ol>
     </td></tr></tbody></table>
     
     <div class="text">
     <h4 class="section"> 
     2015
     </h4></div>
     <table cellpadding="7" cellspacing="7" class="paper">
     <tbody><tr><td>
     <ol>
     <li class="text" value=37>
     <p class='text-justify'>
     <i> Tian R, Basu MK, <span style="font-weight:500;">Capriotti E*</span>.</i> (2015).
     Computational methods and resources for the interpretation of genomic variants in cancer.
     <b>BMC Genomics.</b> 16 (Suppl. 8): S7.<br>
     <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/26111056"><img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/Tian_BMCGenomics2015.pdf"><img width="15" src="pages/img/pdf.gif"></a>
     <a href="pages/documents/papers/Tian_BMCGenomics2015-supmat.pdf">
     <img width="15" src="pages/img/doc.gif"></a>
      &nbsp;
      <span class="__dimensions_badge_embed__" data-doi="10.1186/1471-2164-16-S8-S7" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
      <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1186/1471-2164-16-S8-S7" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
     <br>

     <li class="text" value=36>
     <p class='text-justify'>
     <i>Beerten J, Van Durme J, Gallardo R, <span style="font-weight:500;">Capriotti E</span>, Serpell L, Rousseau F, Schymkowitz J.</i> (2015).
     WALTZ-DB: a benchmark database of amyloidogenic hexapeptides.
     <b>Bioinformatics.</b> 31:1698-1700.<br>
     <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/25600945"><img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/Beerten_Bioinformatics2015.pdf"><img width="15" src="pages/img/pdf.gif"></a>
      &nbsp;
      <span class="__dimensions_badge_embed__" data-doi="10.1093/bioinformatics/btv027" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
      <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1093/bioinformatics/btv027" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
     <br></ol>
     </td></tr></tbody></table>
     <div class="text">
     
     <h4 class="section"> 
     2014
     </h4></div>
     <table cellpadding="7" cellspacing="7" class="paper">
     <tbody><tr><td>
     <ol>
     <li class="text" value=35>
     <p class='text-justify'>
     <i> Tian R, Basu MK, <span style="font-weight:500;">Capriotti E*</span>.</i> (2014).
     ContrastRank: a new method for ranking putative cancer driver genes and classification of tumor samples.
     <b>Bioinformatics.</b>  30: i572-i578.<br>
     <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/25161249"><img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/Tian_Bioinformatics2014.pdf"><img width="15" src="pages/img/pdf.gif"></a>
     <a href="pages/documents/papers/Tian_Bioinformatics2014_SupMat.pdf">
     <img width="15" src="pages/img/doc.gif"></a>
     <a target=blank href="https://snps.biofold.org/contrastrank"><img width="18" src="pages/img/webserver.jpg"></a>
      &nbsp;
      <span class="__dimensions_badge_embed__" data-doi="10.1093/bioinformatics/btu466" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
      <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1093/bioinformatics/btu466" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
     <br>

     <li class="text" value=34>
     <p class='text-justify'>
     <i> Di Tommaso P, Bussotti G, Kemena C, <span style="font-weight:500;">Capriotti E</span>, Chatzou M, Prieto Barja P, Notredame C.</i> (2014).
     SARA-Coffee web server, a tool for the computation of RNA sequence and structure multiple alignments.
     <b>Nucleic Acids Research.</b> 42: W356-360<br>
     <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/24972831"><img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/DiTommaso_NAR2014.pdf"><img width="15" src="pages/img/pdf.gif"></a>
     <a target=blank href="https://tcoffee.crg.eu/apps/tcoffee/do:saracoffee"><img width="18" src="pages/img/webserver.jpg"></a>
     &nbsp;
     <span class="__dimensions_badge_embed__" data-doi="10.1093/nar/gku459" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
     <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1093/nar/gku459" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
     <br>

     <li class="text" value=33>
     <p class='text-justify'>
     <i> Li B, Seligman C, Thusberg J, Miller JL, Auer J, Whirl-Carrillo M, <span style="font-weight:500;">Capriotti E</span>, Klein TE, Mooney SD.</i> (2014).
     In silico comparative characterization of pharmacogenomic missense variants.
     <b>BMC Genomics.</b> 15 (Suppl. 4): S4.<br>
     <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/25057096"><img src="pages/img/pubmed.gif"></a>
     <a href="pages/documents/papers/Li_BMCGenomics2014.pdf"><img width="15" src="pages/img/pdf.gif"></a>
     &nbsp;
     <span class="__dimensions_badge_embed__" data-doi="10.1186/1471-2164-15-S4-S4" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
     <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1186/1471-2164-15-S4-S4" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
     <br></ol>
     </td></tr></tbody></table>

     <div class="text">
     <h4 class="section"> 
     2013
     </h4></div>
    <table cellpadding="7" cellspacing="7" class="paper">
      <tbody><tr><td>
    <ol> 
        <li class="text" value=32>
        <p class='text-justify'>
        <i>Compiani M<sup>&dagger;</sup>*, <span style="font-weight:500;">Capriotti E<sup>&dagger;</sup>*</span>.</i> (2013).
        Computational and theoretical methods for protein folding. 
        <b>Biochemistry.</b> 52: 8601-8624.<br>
        <a target="blank" href="http://www.ncbi.nlm.nih.gov/pubmed/24187909"><img src="pages/img/pubmed.gif" /></a> 
        <a href="pages/documents/papers/Compiani_Biochemistry2013.pdf"><img width="15" src="pages/img/pdf.gif" /></a>
        &nbsp;
         <span class="__dimensions_badge_embed__" data-doi="10.1021/bi4001529" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
         <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1021/bi4001529" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
        <br> 
    <li class="text" value=31>
    <p class='text-justify'>
        <i>Norambuena T, Cares JF, <span style="font-weight:500;">Capriotti E</span>, Melo F</i> (2013).
        WebRASP: a server for computing energy scores to assess 
        the accuracy and stability of RNA 3D structures. 
        <b>Bioinformatics.</b> 29:2649-2650.<br>
        <a target="blank" href="http://www.ncbi.nlm.nih.gov/pubmed/23929030"><img src="pages/img/pubmed.gif" /></a> 
        <a href="pages/documents/papers/Norambuena_Bioinformatics2013.pdf"><img width="15" src="pages/img/pdf.gif" /></a>
        <a target=blank href="http://melolab.org/webrasp"><img width="18" src="pages/img/webserver.jpg"></a>
        &nbsp;
         <span class="__dimensions_badge_embed__" data-doi="10.1093/bioinformatics/btt441" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
         <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1093/bioinformatics/btt441" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
        <br> 

    <li class="text" value=30>
    <p class='text-justify'>
            <i><span style="font-weight:500;">Capriotti E*</span>, Calabrese R, Fariselli P,  Martelli PL, Altman RB,
         Casadio R*</i> (2013).
        WS-SNPs&GO: a web server for predicting the deleterious effect of 
        human protein variants using functional annotation. 
	<b>BMC Genomics.</b> 14 Suppl 3:S6.<br>
        <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/23819482">
        <img src="pages/img/pubmed.gif"></a>
        <a href="pages/documents/papers/Capriotti_SNPsGO_BMCGenomics13.pdf">
        <img width="15" src="pages/img/pdf.gif"></a>
        <a target=blank href="https://snps.biofold.org/snps-and-go"><img width="18" src="pages/img/webserver.jpg"></a>
        <a target=blank href="https://hub.docker.com/r/biofold/snps-and-go"><img width="24" src="pages/img/docker.png"></a>
	&nbsp;
         <span class="__dimensions_badge_embed__" data-doi="10.1186/1471-2164-14-S3-S6" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
         <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1186/1471-2164-14-S3-S6" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
        <br>

    <li class="text" value=29>
    <p class='text-justify'>
        <i><span style="font-weight:500;">Capriotti E*</span>, Altman RB, Bromberg Y*</i> (2013).
        Collective judgment predicts disease-associated single nucleotide variants.
        <b>BMC Genomics.</b> 14 Suppl 3:S2.<br>
        <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/23819846">
        <img src="pages/img/pubmed.gif"></a>
        <a href="pages/documents/papers/Capriotti_MetaSNP_BMCGenomics13.pdf">
        <img width="15" src="pages/img/pdf.gif"></a>
        <a href="pages/documents/papers/Capriotti_MetaSNP_BMCGenomics13-supmat.pdf">
     <img width="15" src="pages/img/doc.gif"></a>
        <a target=blank href="https://snps.biofold.org/meta-snp"><img width="18" src="pages/img/webserver.jpg"></a>
        <a target=blank href="https://hub.docker.com/r/biofold/meta-snp"><img width="24" src="pages/img/docker.png"></a>
        &nbsp;
	<span class="__dimensions_badge_embed__" data-doi="10.1186/1471-2164-14-S3-S2" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
	<span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1186/1471-2164-14-S3-S2" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
         <br>

	<li class="text" value=28>
	<p class='text-justify'>
    <i> Kemena C, Bussotti G, <span style="font-weight:500;">Capriotti E</span>, Marti-Renom MA, Notredame C.</i> (2013).
    Using tertiary structure for the computation of highly accurate multiple RNA alignments with the
    SARA-Coffee package.
        <b>Bioinformatics.</b> 29:1112-1119.<br>
        <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/23449094">
    <img src="pages/img/pubmed.gif"></a>
    <a href="pages/documents/papers/Kemena_Bioinformatics2013.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
    <a href="pages/documents/papers/Kemena_Bioinformatics2013-supmat.pdf">
     <img width="15" src="pages/img/doc.gif"></a>
    &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1093/bioinformatics/btt096" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1093/bioinformatics/btt096" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br></ol>
    </td></tr></tbody></table>
    
     <div class="text">
     <h4 class="section"> 
     2012
     </h4></div>
    <table cellpadding="7" cellspacing="7" class="paper">
      <tbody><tr><td>
    <ol>
	<li class="text" value=27>
	<p class='text-justify'>
    <i> Lahti JL, Tang GW, <span style="font-weight:500;">Capriotti E</span>, Liu T, Altman RB.</i> (2012).
    Bioinformatics and variability in drug response: a protein structural perspective.
        <b>J R Soc Interface.</b> 9; 1409-1437.<br>
        <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/22552919">
    <img src="pages/img/pubmed.gif"></a>
    <a href="pages/documents/papers/Lahti_JRSI2012.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
    &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1098/rsif.2011.0843" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1098/rsif.2011.0843" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br>

    <li class="text" value=26>
    <p class='text-justify'>
    <i><span style="font-weight:500;">Capriotti E<sup>&dagger;</sup>*</span>, Nehrt NL<sup>&dagger;</sup>, Kann MG<sup>&dagger;</sup>*, Bromberg Y<sup>&dagger;</sup>*.</i> (2012).
    Bioinformatics for personal genome interpretation. 
    <b>Briefings in Bioinformatics</b>. 13; 495-512.<br> 
    <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/22247263">
    <img src="pages/img/pubmed.gif"></a>
    <a target=blank href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3404395/pdf/bbr070.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
       &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1093/bib/bbr070" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1093/bib/bbr070" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br></ol>
    </td></tr></tbody></table>

     <div class="text">
     <h4 class="section"> 
     2011
     </h4></div>
    <table cellpadding="7" cellspacing="7" class="paper">
      <tbody><tr><td>
    <ol>
    <li class="text" value=25>
    <p class='text-justify'>
    <i> Dewey FE, Chen R, Cordero SP, Ormond KE, Caleshu C, Karczewski KJ,
        Whirl-Carrillo M, Wheeler MT, Dudley JT, Byrnes JK, Cornejo OE, 
        Knowles JW, Woon M, Sangkuhl K, Gong L, Thorn CF, Hebert JM, 
        <span style="font-weight:500;">Capriotti E</span>, David SP, Pavlovic A, West A, Thakuria JV, Ball MP, 
        Zaranek AW, Rehm HL, Church GM, West JS, Bustamante CD, Snyder M, 
        Altman RB, Klein TE, Butte AJ, Ashley EA.</i> (2011). 
        Phased whole-genome genetic risk in a family quartet using a major 
        allele reference sequence. 
        <b>PLOS Genetics.</b> 7; e1002280.<br>
	<a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/21935354">
    <img src="pages/img/pubmed.gif"></a>
    <a href="pages/documents/papers/Dewey_PLOSGenetics2011.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
    &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pgen.1002280" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1371/journal.pgen.1002280" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br>

    <li class="text" value=24>
    <p class='text-justify'>
    <i><span style="font-weight:500;">Capriotti E*</span>, Altman RB.</i> (2011).
    A new disease-specific machine learning approach for the prediction 
    of cancer-causing missense variants. 
    <b>Genomics.</b> 98; 310-317.<br>
    <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/21763417">
    <img src="pages/img/pubmed.gif"></a>
    <a href="pages/documents/papers/Capriotti_Genomics2011.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
    <a href="pages/documents/papers/Capriotti_Genomics2011-supmat.pdf">
     <img width="15" src="pages/img/doc.gif"></a>
    <a target=blank href="https://snps.biofold.org/drcancer"><img width="18" src="pages/img/webserver.jpg"></a>
     &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1016/j.ygeno.2011.06.010" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1016/j.ygeno.2011.06.010" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br>

    <li class="text" value=23>
    <p class='text-justify'>
    <i><span style="font-weight:500;">Capriotti E*</span>, Altman RB.</i> (2011).
    Improving the prediction of disease-related variants using protein 
    three-dimensional structure.
    <b>BMC Bioinformatics.</b> 12 (Suppl 4); S3<br>
    <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/21992054">
    <img src="pages/img/pubmed.gif"></a>
    <a href="pages/documents/papers/Capriotti_AIMM_BMCBioinformatics2011.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
     &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1186/1471-2105-12-S4-S3" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1186/1471-2105-12-S4-S3" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br>

    <li class="text" value=22>
    <p class='text-justify'>
    <i>Fernald GH, <span style="font-weight:500;">Capriotti E</span>, Daneshjou R, Karczewski KJ, Altman RB.</i> (2011).
       Bioinformatics challenges for personalized medicine.
    <b>Bioinformatics.</b> 27; 1741-1748<br>
    <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/21596790">
    <img src="pages/img/pubmed.gif"></a>
    <a href="pages/documents/papers/Fernald_Bioinformatics2011.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
     &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1093/bioinformatics/btr295" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1093/bioinformatics/btr295" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br>

    <li class="text" value=21>
    <p class='text-justify'>
    <i>Liu T, Tang GW, <span style="font-weight:500;">Capriotti E*</span>.</i> (2011).
       Comparative modeling: the state of the art and protein drug target 
       structure prediction.
    <b>Combinatorial Chemistry & High Throughput Screening.</b> 14: 532-547<br>
    <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/21521153">
    <img src="pages/img/pubmed.gif"></a>
    <a href="pages/documents/papers/Liu_CCHTS2011.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
    &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.2174/138620711795767811" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.2174/138620711795767811" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br>

    <li class="text" value=20>
    <p class='text-justify'>
    <i><span style="font-weight:500;">Capriotti E<sup>&dagger;</sup></span>, Norambuena T<sup>&dagger;</sup>, Marti-Renom MA, Melo F. </i> (2011). 
    All atom knowledge-based potential for RNA structure prediction and assessment. 
    <b>Bioinformatics</b>. 27; 1086-1093.<br>
    <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/21349865">
    <img src="pages/img/pubmed.gif"></a>
    <a href="pages/documents/papers/Capriotti_Bioinformatics2011.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
     <a href="pages/documents/papers/Capriotti_Bioinformatics2011-supmat.pdf">
     <img width="15" src="pages/img/doc.gif"></a>
    &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1093/bioinformatics/btr093" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>   
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1093/bioinformatics/btr093" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br> 
    <br>

    <li class="text" value=19>
    <p class='text-justify'>
    <i>Bau D, Sanyal A, Lajoie BR, <span style="font-weight:500;">Capriotti E</span>, Byron M, Lawrence JB, Dekker J, Marti-Renom MA. </i> (2011). The three-dimensional folding of the alpha-globin    gene domain reveals formation of chromatin globules.
    <b>Nat Struct Mol Biol</b>. 18: 107-114.<br>
    <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/21131981">
    <img src="pages/img/pubmed.gif"></a>
    <a href="pages/documents/papers/Bau_NSMB2011.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
    <a href="pages/documents/papers/Bau_NSMB2011-supmat.pdf">
     <img width="15" src="pages/img/doc.gif"></a>
    &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1038/nsmb.1936" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1038/nsmb.1936" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br></ol>
    </td></tr></tbody></table>

     <div class="text">
     <h4 class="section"> 
     2010
     </h4></div>
    <table cellpadding="7" cellspacing="7" class="paper">
      <tbody><tr><td>
    <ol>
    <li class="text" value=18>
    <p class='text-justify'>
    <i><span style="font-weight:500;">Capriotti E</span>, Marti-Renom MA.</i> (2010). Quantifying the relationship 
    between sequence and three-dimensional structure conservation in RNA. 
    <b>BMC Bioinformatics</b>. 11: 322.<br> 
    <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/20550657">
    <img src="pages/img/pubmed.gif"></a>
    <a href="pages/documents/papers/Capriotti_BMCBioinfo2010.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
    <a href="pages/documents/papers/Capriotti_BMCBioinfo2010-supmat.pdf">
     <img width="15" src="pages/img/doc.gif"></a>
    &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1186/1471-2105-11-322" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1186/1471-2105-11-322" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br></ol>
    </td></tr></tbody></table>

     <div class="text">
     <h4 class="section"> 
     2009
     </h4></div>
    <table cellpadding="7" cellspacing="7" class="paper">
      <tbody><tr><td>
    <ol>
    <li class="text" value=17>
    <p class='text-justify'>
    <i>Calabrese R, <span style="font-weight:500;">Capriotti E</span>, Fariselli P, Martelli PL, Casadio R.</i> 
    (2009). Functional annotations improve the predictive score of human 
    disease-related mutations in proteins. <b>Human Mutation</b>. 
    30: 1237-1244.<br>
    <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/19514061">
    <img src="pages/img/pubmed.gif"></a>
    <a href="pages/documents/papers/Calabrese_HumMut09.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
    &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1002/humu.21047" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1002/humu.21047" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br>

    <li class="text" value=16>
    <p class='text-justify'>
    <i><span style="font-weight:500;">Capriotti E</span>, Marti-Renom MA.</i> (2009). SARA: a server for function 
    annotation of RNA structures. <b>Nucleic Acids Research</b>. 37
    (Web Server issue): W260-W265.<br>
    <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/19483098">
    <img src="pages/img/pubmed.gif"></a>
    <a href="pages/documents/papers/CapriottiNAR09.pdf">
    <img width="15" src="pages/img/pdf.gif"></a> 
    <a target=blank href="https://structure.biofold.org/sara"><img width="18" src="pages/img/webserver.jpg"></a>
     &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1093/nar/gkp433" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1093/nar/gkp433" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    </td></tr>
    
    </tbody></table> 

     <div class="text">
     <h4 class="section"> 
     2008
     </h4></div>
    <table cellpadding="7" cellspacing="7" class="paper">
      <tbody><tr><td>
    <ol>
    <li class="text" value=15>
    <p class='text-justify'>
    <i><span style="font-weight:500;">Capriotti E</span>, Marti-Renom MA.</i> (2008). RNA structure alignment by 
    a unit-vector approach.
    <b>Bioinformatics</b>.  24: i112-i116.<br>
    <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/18689811">
    <img src="pages/img/pubmed.gif"></a>
    <a href="pages/documents/papers/Capriotti_ECCB08.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
    &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1093/bioinformatics/btn288" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1093/bioinformatics/btn288" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br>

    <li class="text" value=14>
    <p class='text-justify'>
    <i><span style="font-weight:500;">Capriotti E</span>, Fariselli P, Rossi I, Casadio R.</i> (2008). A three-state 
    prediction of single point mutations on protein stability changes. 
    <b>BMC Bioinfomatics</b>. 9 (Suppl 2): S6.<br>
    <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/18387208">
    <img src="pages/img/pubmed.gif"></a>
    <a href="pages/documents/papers/Capriotti_BMCBioinfo08.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
     &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1186/1471-2105-9-S2-S6" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1186/1471-2105-9-S2-S6" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br>

    <li class="text" value=13>
    <p class='text-justify'>
    <i><span style="font-weight:500;">Capriotti E</span>, Marti-Renom MA.</i> (2008). Computational RNA structure 
    prediction. <b>Current Bioinformatics</b>. 3: 32-45.<br>
    <a target=blank href="http://benthamscience.com/journal/abstracts.php?journalID=cbio&articleID=82019">
    <img src="pages/img/abstract.gif"></a>
    <a href="pages/documents/papers/Capriotti_CBio08.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
     &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.2174/157489308783329823" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.2174/157489308783329823" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br>
    
    <li class="text" value=12>
    <p class='text-justify'>
    <i><span style="font-weight:500;">Capriotti E</span>, Arbiza L, Casadio R, Dopazo J, Dopazo H, Marti-Renom MA.</i>
    (2008). The use of estimated evolutionary strength at the codon level 
    improves the prediction of disease related protein mutations in human. 
    <b>Human Mutation.</b> 29: 198-204.<br>
    <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/17935148">
    <img src="pages/img/pubmed.gif"></a>
    <a href="pages/documents/papers/Capriotti_HumMut08.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
    &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1002/humu.20628" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1002/humu.20628" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br></ol>
    </td></tr></tbody></table>

     <div class="text">
     <h4 class="section"> 
     2007
     </h4></div>
    <table cellpadding="7" cellspacing="7" class="paper">
      <tbody><tr><td>
    <ol>
    <li class="text" value=11>
    <p class='text-justify'>
    <i><span style="font-weight:500;">Capriotti E*</span>, Casadio R.</i> (2007). K-Fold: a tool for the prediction of
    the protein folding kinetic order and rate.
    <b>Bioinformatics</b>. 23: 385-386.<br>
    <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/17138584">
    <img src="pages/img/pubmed.gif"></a>
    <a href="pages/documents/papers/KFold_Bioinfo07.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
    <a target=blank href="https://folding.biofold.org/k-fold"><img width="18" src="pages/img/webserver.jpg"></a>
     &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1093/bioinformatics/btl610" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1093/bioinformatics/btl610" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br>

    <li class="text" value=10>
    <p class='text-justify'>
    <i>Fariselli P, Rossi I, <span style="font-weight:500;">Capriotti E</span>, Casadio R.</i> (2007) The WWWH of 
    remote homolog detection: The state of the art. 
    <b>Brief. Bioinformatics</b>. 8: 78-87.<br>
    <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/17003074">
    <img src="pages/img/pubmed.gif"></a>
    <a href="pages/documents/papers/WWWH_BriefBioinf_06.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
    &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1093/bib/bbl032" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
        <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1093/bib/bbl032" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br></ol>
    </td></tr></tbody></table>

     <div class="text">
     <h4 class="section"> 
     2006
     </h4></div>
    <table cellpadding="7" cellspacing="7" class="paper">
      <tbody><tr><td>
    <ol>
    <li class="text" value=9>
    <p class='text-justify'>
    <i><span style="font-weight:500;">Capriotti E</span>, Compiani M.</i> (2006). Diffusion-collision of foldons 
    elucidates the kinetic effects of point mutations and suggests control 
    strategies of the folding process of helical proteins. 
    <b>Proteins</b>. 64: 198-209.<br>
    <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/16609971">
    <img src="pages/img/pubmed.gif"></a>
    <a href="pages/documents/papers/ProtMut06.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
    &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1002/prot.20980" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1002/prot.20980" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br>

    <li class="text" value=8>
    <p class='text-justify'>
    <i><span style="font-weight:500;">Capriotti E</span>, Calabrese R, Casadio R.</i> (2006). Predicting the 
    insurgence of human genetic diseases associated to single point protein 
    mutations with support vector machines and evolutionary information. 
    <b>Bioinformatics</b>. 22: 2729-2734.<br>
    <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/16895930">
    <img src="pages/img/pubmed.gif"></a>
    <a href="pages/documents/papers/BioinfDis06.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
    <a target=blank href="https://snps.biofold.org/phd-snp"><img width="18" src="pages/img/webserver.jpg"></a>
    &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1093/bioinformatics/btl423" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1093/bioinformatics/btl423" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br>
    
    <li class="text" value=7>
    <p class='text-justify'>
    <i>Grandi F, Sandal M, Guarguaglini G, <span style="font-weight:500;">Capriotti E</span>, Casadio R, Samori B.</i>
    Hierarchical Mechanochemical Switches in Angiostatin.
    <b>Chembiochem</b>. 7: 1774-1782.<br>
    <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/16991168">
    <img src="pages/img/pubmed.gif"></a>
    <a href="pages/documents/papers/ChemBiochem06.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
    <a href="pages/documents/papers/ChemBiochem06-supmat.pdf">
     <img width="15" src="pages/img/doc.gif"></a>
    &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1002/cbic.200600227" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span><span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1002/cbic.200600227" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br></ol>
    </td></tr></tbody></table>

     <div class="text">
     <h4 class="section"> 
     2005
     </h4></div>
    <table cellpadding="7" cellspacing="7" class="paper">
    <tbody><tr><td>
    <ol>
    <li class="text" value=6>
    <p class='text-justify'>
    <i>Stizza A, <span style="font-weight:500;">Capriotti E</span>, Compiani M. </i> (2005). A Minimal Model of 
    Three-State Folding Dynamics of Helical Proteins.
    <b>J. Phys. Chem. B</b>. 109: 4215-4226.<br>
    <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/16851484">
    <img src="pages/img/pubmed.gif"></a>
    <a href="pages/documents/papers/jcpB05.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
    &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1021/jp045228d" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1021/jp045228d" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br>
    
    <li class="text" value=5>
    <p class='text-justify'>
    <i><span style="font-weight:500;">Capriotti E</span>, Fariselli P, Casadio R.</i> (2005). I-Mutant2.0: predicting
     stability changes upon mutation from the protein sequence or structure.
    <b>Nucleic Acids Research</b>. 33(Web Server issue): W306-W310.<br>
    <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/15980478">
    <img src="pages/img/pubmed.gif"></a>
    <a href="pages/documents/papers/nar05.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
   <a target=blank href="https://snps.biofold.org/i-mutant"><img width="18" src="pages/img/webserver.jpg"></a>
    &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1093/nar/gki375" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1093/nar/gki375" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br>
    
    <li class="text" value=4>
    <p class='text-justify'>
    <i><span style="font-weight:500;">Capriotti E</span>, Fariselli P, Calabrese R, Casadio R.</i> (2005) Predicting 
    protein stability changes from sequences using support vector machines.   
    <b>Bioinformatics</b>.  21: ii54-ii58.<br>
    <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/16204125">
    <img src="pages/img/pubmed.gif"></a>
    <a href="pages/documents/papers/ECCB05.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
    &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1093/bioinformatics/bti1109" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1093/bioinformatics/bti1109" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br></ol>
    </td></tr></tbody></table>

     <div class="text">
     <h4 class="section"> 
     2004
     </h4></div>
    <table cellpadding="7" cellspacing="7" class="paper">
    <tbody><tr><td>
    <ol>
    <li class="text" value=3>
    <p class='text-justify'>
    <i><span style="font-weight:500;">Capriotti E</span>, Fariselli, Casadio R</i> (2004). A neural network-based 
    method for predicting protein stability changes upon single point mutations.
    <b>Bioinformatics</b>. 20 (Suppl 1): I63-I68<br>
    <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/15262782">
    <img src="pages/img/pubmed.gif"></a>
    <a href="pages/documents/papers/ISMB04.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
    &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1093/bioinformatics/bth928" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1093/bioinformatics/bth928" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br>
    
    <li class="text" value=2>
    <p class='text-justify'>
    <i>Compiani M, <span style="font-weight:500;">Capriotti E</span>, Casadio R.</i> (2004). The Dynamics 
    of the Minimally Frustrated Helices Determine the Hierarchical Folding 
    of Small Helical Proteins.
    <b>Physical Review E</b>. 69: 051905<br>
    <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/15244845">
    <img src="pages/img/pubmed.gif"></a>
    <a href="pages/documents/papers/PysRevE.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
    &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1103/PhysRevE.69.051905" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1103/PhysRevE.69.051905" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br>
    
    <li class="text" value=1>
    <p class='text-justify'>
    <i><span style="font-weight:500;">Capriotti E</span>, Fariselli P, Rossi I, Casadio R</i> (2004). A Shannon
    entropy-based filter detects high-quality profile-profile alignments in
    searches for remote homologues.
    <b>Proteins</b>. 54: 351-360.<br>
    <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/14696197">
    <img src="pages/img/pubmed.gif"></a>
    <a href="pages/documents/papers/Proteins04.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
    &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1002/prot.10564" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1002/prot.10564" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    </td></tr></tbody></table>


     <br>
     <div class="text">
     <h4 class="section"> 
     Book chapters
     </h4></div>
   <table cellpadding="7" cellspacing="7" class="paper">
    <tbody><tr><td>
    <ol>
    <li class="text" value=6>
    <p class='text-justify'>
    <i>Dufour D, <span style="font-weight:500;">Capriotti E</span>, Marti-Renom MA.</i> (2014). 
    Computational methods for RNA structure prediction and analysis. 
    <b>RNA Nanotechnology</b>. CRC Press (Wang B Editor), 21-50.
    <br><a target=blank href="http://www.panstanford.com/books/9789814411646.html">
    <img src="pages/img/abstract.gif"></a> 
    <a href="http://books.google.com/books?id=NB0yAwAAQBAJ&printsec=frontcover&source=gbs_ge_summary_r&cad=0#v=onepage&q&f=false"><img src="pages/img/google.png"></a>
    <a href="pages/documents/papers/RNA_Nanotechnology_Chapter_3.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
    &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1201/b15533-5" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1201/b15533-5" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br><br>

    <li class="text" value=5>
    <p class='text-justify'>
    <i><span style="font-weight:500;">Capriotti E</span>.</i> (2013). Comparative modeling and structure prediction:
    application to drug discovery.
    <b>In silico drug discovery and design</b>.
    Future Science (Lill MA Editor), 34-48.<br><a target=blank href="http://www.futuremedicine.com/doi/abs/10.4155/ebo.13.192">
    <img src="pages/img/abstract.gif"></a>
    <a href="pages/documents/papers/capriotti_futsci2013.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
    &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.4155/ebo.13.192" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.4155/ebo.13.192" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br>

    <li class="text" value=4>
    <p class='text-justify'>
    <i>Marti-Renom MA, <span style="font-weight:500;">Capriotti E</span>, Shindyalov I, Bourne P.</i> (2009).
    Structural Comparison and Alignment.<b> Structural Bioinformatics</b> 
    II Edition. John Wiley & Sons, Inc. (Gu J and Bourne PE Editors), 397-418.<br>
    <a target=blank href="http://www.wiley.com/WileyCDA/WileyTitle/productCd-0470181052.html">
    <img src="pages/img/abstract.gif"></a>
    <a target=blank href="http://www.wiley.com/WileyCDA/WileyTitle/productCd-0470181052,descCd-google_preview.html">
    <img src="pages/img/google.png"></a>
    <a href="pages/documents/papers/Marti-Renom_StructuralBioinformatics2009.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
    <br><br>

    <li class="text" value=3>
    <p class='text-justify'>
    <i><span style="font-weight:500;">Capriotti E</span>, Marti-Renom, MA.</i> (2007). Assessment of protein structure
    predictions. <b>Computational Structural Biology: Methods and 
    Applications</b>. World Scientific Publishing Company 
    (Schwede T and Peitsch MC Editors), 89-109.<br><a target=blank href="http://www.worldscientific.com/worldscibooks/10.1142/6659">
    <img src="pages/img/abstract.gif"></a>
    <a href="pages/documents/papers/Capriotti_TorstenBook08.pdf"><img width="15" src="pages/img/pdf.gif"></a>
    &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1142/9789812778789_0004" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1142/9789812778789_0004" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br>


    <li class="text" value=2>
    <p class='text-justify'>
    <i>Bartoli L, <span style="font-weight:500;">Capriotti E</span>, Fariselli P, Martelli PL, Casadio R.</i> (2007).
    The pros and cons of predicting protein contact maps. <b>Protein Structure 
    Prediction Series: Methods in Molecular Biology</b>. Springer, Humana 
    Press, (Zaki M and Bystroff C Editors) 2nd ed., 413: 199-218.<br>
    <a target=blank href="http://www.ncbi.nlm.nih.gov/pubmed/18075167">
    <img src="pages/img/pubmed.gif"></a>
    <a href="pages/documents/papers/Bartoli_MethMolBiol2008.pdf">
    <img width="15" src="pages/img/pdf.gif"></a>
    &nbsp;
    <span class="__dimensions_badge_embed__" data-doi="10.1007/978-1-59745-574-9_8" data-hide-zero-citations="false" data-style="small_rectangle" style="display:inline-block;"></span>
    <span style="display:inline-block;"><a href="https://plu.mx/plum/a/?doi=10.1007/978-1-59745-574-9_8" class="plumx-plum-print-popup" data-size="tiny" data-popup="right" data-badge="true"  data-hide-when-empty="false"></a> </span>
    <br>
    <br>

    <li class="text" value=1>
    <p class='text-justify'>
    <i>Casadio R, <span style="font-weight:500;">Capriotti E</span>, Compiani M, Fariselli P, Jacoboni I, Martelli PL 
    Rossi I, Tasco G.</i> (2003). Neural Networks and the Prediction of Protein
    Structure. In: Artificial Intelligence and Heuristic Methods 
    for Bioinformatics (Frasconi P and Shamir R Editors) 
    <b>NATO Series Books. IOS Press Amsterdam</b>, 22-33.
    <br><a target=blank href="http://www.iospress.nl/book/artificial-intelligence-and-heuristic-methods-in-bioinformatics/">
    <img src="pages/img/abstract.gif"></a>
   <br></ol>
   </td></tr>
   <tr><td>
   

     <br>
     <div class="text">
     <h4 class="section"> 
     Citations
     </h4></div>
     <p class='text-justify'>
   <ul class="text">
    <li>
    <a href="http://scholar.google.es/citations?user=qAa7Et0AAAAJ" target=blank>Google Scholar</a>
    </li>
   <br>
    <li>
    <a href="http://www.scopus.com/authid/detail.url?authorId=8851983500" target=blank>Scopus Preview</a>
    </li>
   <br>
    <li>
    <a href="http://www.researcherid.com/rid/D-9318-2011" target=blank>ResearcherID</a>
    </li>
   </ul>
   </p>
   </td></tr>
   </tbody></table>
   <br><br>    
   <div class='footer-ok'><div>

      </div>
    </div> <!-- /container -->


    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug
    <script src="pages/assets/js/ie10-viewport-bug-workaround.js"></script>
    -->
  <!-- PlumX -->
    <script type="text/javascript" src="//cdn.plu.mx/widget-popup.js"></script>
  </body>
</html>
